x-pack/test/functional/services/ml/ml_nodes_list.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export function MlNodesPanelProvider({ getService }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+
+  return {
+    async assertNodesOverviewPanelExists(expectPanelExits: boolean = true) {
+      if (expectPanelExits) {
+        await testSubjects.existOrFail('mlNodesOverviewPanel');
+      } else {
+        await testSubjects.missingOrFail('mlNodesOverviewPanel');
+      }
+    },
+
+    async assertNodesListLoaded() {
+      await testSubjects.existOrFail('mlNodesTable loaded', { timeout: 5000 });
+    },
+
+    async assertMlNodesCount(expectedCount: number = 1) {
```

Comment:
Nit: I'd suggest to rename the parameter to something like `minCount` to indicate the `>=` check.

Reply:
renamed in https://github.com/elastic/kibana/pull/124793/commits/3faa423099326a7811d3a801d39554f0558f488b